lib/commands.js

```diff
@@ -436,6 +437,12 @@ module.exports = async (message, command, args) => {
         embed: embed
       })
     }
+    case 'ws': {
+      const wikiResult = await fetch(`https://oldschool.runescape.wiki/api.php?action=opensearch&search=${value}&limit=1&redirects=resolve`)
+        .then(res => res.json())
+
+      return message.channel.send(!wikiResult[1].length || !wikiResult[3].length ? '**ERROR:**\nSearch did not return any results' : `**${wikiResult[1]}:**\n<${wikiResult[3]}>`)
```

Comment:
I would do this in same format as custom commands, e.g without newline. Also what happens when the wiki do not even returns array, you should also check for that.